src/app/components/audio-analysis/pages/list/list.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { ListComponent } from "./list.component";
+
+xdescribe("ListComponent", () => {
+  let component: ListComponent;
+  let fixture: ComponentFixture<ListComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ListComponent],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ListComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it("should create", () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
pointless test?

if it's just testing costruction, at least make the `toBeTruthy` expectation more useful. Why not test that component is an instance of AudioAnalysis.ListComponent ?

Reply:
Another good test for these pages is asserting the `baw-client` element/component is exists/rendered in the template

Reply:
This test is disabled, (`xdescribe`). It is implemented later on